workflows/diagnostics/fv3net/diagnostics/offline/_input_sensitivity.py

```diff
@@ -22,6 +22,11 @@ def _stack_sample_data(ds: xr.Dataset) -> xr.Dataset:
         stack_dims = ["tile", "x", "y", DATASET_DIM_NAME]
     else:
         stack_dims = ["tile", "x", "y"]
+    if "ncol" in ds.dims:
```

Comment:
Is there a way to pass the horizontal dims to this function? So could just modify it to be `stack_dims = horizontal_dims + [DATASET_DIM_NAME]` etc. instead of needing a new if statement.

Reply:
Done. Needs to be sorted because `horizontal_dims` is `["x","y","tile"]` but here it needs to be tile first.

Reply:
Probably would be fine to just define `horizontal_dims` with the tile-first order.